crates/ty_python_semantic/src/types/signatures.rs

```diff
@@ -917,10 +917,12 @@ impl<'db> Signature<'db> {
                     ParameterKind::PositionalOrKeyword {
                         name: self_name,
                         default_type: self_default,
```

Comment:
do we even need to keep the `default_type` field if we now have a `default_value` field? I think we literally only use the `default_type` for rendering in display; I think it's completely irrelevant for type-checking purposes. (For type-checking purposes, all you need to know is whether a parameter _has_ a default; you don't need to know what the default _is_.)